pmm/pmm2-server-release.groovy

```diff
@@ -159,7 +160,7 @@ pipeline {
                 withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                     sh """
                         sg docker -c "
-                            docker login -u "${USER}" -p "${PASS}"
+                            echo "${PASS}" | docker login -u "${USER}" --password-stdin
```

Comment:
would not it print $PASS into log? if yes it is security hole!

Reply:
Another question - why it worked correctly for pmm-server but failed to push pmm-client?

Reply:
> Another question - why it worked correctly for pmm-server but failed to push pmm-client?

Because, one and the same account may have different permissions on different repositories. This was our case, and @edux helped resolve it.

Reply:
> would not it print $PASS into log? if yes it is security hole!

>  Using STDIN prevents the password from ending up in the shell’s history, or log-files.

It's quote from here https://docs.docker.com/engine/reference/commandline/login/#provide-a-password-using-stdin